src/routes/webhook/topgg.ts

```diff
@@ -37,7 +43,10 @@ export class UserRoute extends Route {
 	private async voteProcess(provider: VoteProvider, userId: string) {
 		const providerInfo = this.getProviderInfo(provider);
 
-		const guild = await getGuildInformation(GuildIDEnum.BIRTHDAYY_HQ);
+		const guild = await resolveOnErrorCodesDiscord(
```

Comment:
since this is a secario, that we often will face (fetching discord information) i would suggest creating a seperate function for this.